### PR TITLE
Finalbuild skills cleanup

### PR DESCRIPTION
**Pull request by aim2kill**

### CODE REVIEW

File: libs/SoloPlay/BuildFiles/sorceress.BlovaBuild.js [hunk 1]
```diff
@@ -21,15 +21,7 @@ const finalBuild = {
...
-		[sdk.skills.IceBolt, 1],
-		[sdk.skills.IceBlast, 1],
-		[sdk.skills.StaticField, 1],
-		[sdk.skills.Telekinesis, 1],
-		[sdk.skills.FrostNova, 1],
 		[sdk.skills.Teleport, 1],
-		[sdk.skills.GlacialSpike, 1],
-		[sdk.skills.Blizzard, 1],
-		[sdk.skills.ColdMastery, 1],
 		[sdk.skills.Nova, 20],
```
Comment (theBGuy) [inline libs/SoloPlay/BuildFiles/sorceress.BlovaBuild.js:25]: Should stick at least 1 point into Blizz to ensure having it before moving to max out other skills

File: libs/SoloPlay/Functions/MiscOverrides.js [hunk 2]
```diff
@@ -261,7 +261,7 @@ Misc.checkItemsForImbueing = function () {
 	if (!me.getQuest(sdk.quest.id.ToolsoftheTrade, 1)) return false;
 
 	let items = me.getItemsEx()
-		.filter(item => item.getStat(sdk.stats.NumSockets) === 0 && [sdk.itemquality.Normal, sdk.itemquality.Superior].includes(item.quality));	// Normal Quality & Superior Quality
+	.filter(item => item.getStat(sdk.stats.NumSockets) > 0 && AutoEquip.hasTier(item) && ![sdk.itemquality.Normal, sdk.itemquality.Superior].includes(item.quality))
```
Comment (theBGuy) [inline libs/SoloPlay/Functions/MiscOverrides.js:264]: revert this whole thing back
```
let items = me.getItemsEx()
		.filter(item => item.getStat(sdk.stats.NumSockets) === 0 && [sdk.itemquality.Normal, sdk.itemquality.Superior].includes(item.quality));
```

File: libs/SoloPlay/Functions/MiscOverrides.js [hunk 3]
```diff
@@ -464,7 +464,7 @@ Misc.getSocketables = function (item, itemInfo) {
 
 Misc.checkSocketables = function () {
 	let items = me.getItemsEx()
-		.filter(item => item.getStat(sdk.stats.NumSockets) > 0 && AutoEquip.hasTier(item) && ![sdk.itemquality.Normal, sdk.itemquality.Superior].includes(item.quality)) // Normal Quality & Superior Quality
+		.filter(item => item.getStat(sdk.stats.NumSockets) > 0 && AutoEquip.hasTier(item) && ![item.normal, item.superior].includes(item.quality))
```
Comment (theBGuy) [inline libs/SoloPlay/Functions/MiscOverrides.js:467]: ```[sdk.itemquality.Normal, sdk.itemquality.Superior].includes(item.quality))```
remember what I said about numerical vs boolean.
item.quality === sdk.itemquality.Magic // is boolean its just checking if the values equal each other
[sdk.itemquality.Normal, sdk.itemquality.Superior].includes(item.quality)) is equal to the statement
[2, 3].includes(item.quality) // where item.quality is returning the actual value and we are seeing whether the array includes that value

**Review by theBGuy — COMMENTED** [review]
Check the comments, those sections need to be fixed

File: libs/SoloPlay/BuildFiles/sorceress.BlovaBuild.js (hunk 1 above)
Comment (aim2kill) [reply libs/SoloPlay/BuildFiles/sorceress.BlovaBuild.js:25]: my bad, I did a bad copy/paste

File: libs/SoloPlay/Functions/MiscOverrides.js (hunk 2 above)
Comment (aim2kill) [reply libs/SoloPlay/Functions/MiscOverrides.js:264]: my bad, I did a bad copy/paste

File: libs/SoloPlay/Functions/MiscOverrides.js (hunk 3 above)
Comment (aim2kill) [reply libs/SoloPlay/Functions/MiscOverrides.js:467]: my bad, I did a bad copy/paste

**Pull request merged**